docker/Dockerfile.windows.2004

```diff
@@ -0,0 +1,20 @@
+# escape=`
+
+FROM mcr.microsoft.com/windows/servercore:2004 AS git
+SHELL ["powershell.exe", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+RUN [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12 ; `
+    Invoke-WebRequest -UseBasicParsing https://github.com/git-for-windows/git/releases/download/v2.21.0.windows.1/MinGit-2.21.0-64-bit.zip -OutFile git.zip; `
+    Expand-Archive git.zip -DestinationPath C:\git;
+
+FROM mcr.microsoft.com/powershell:nanoserver-2004
```

Comment:
@luthermonson you're using the powershell image as the base. All the scripts are powershell so there needs to be a 20H2 before you can do anything.

Reply:
ah yes, indeed. you win so much copy pasta i didnt even notice. that's a shame but for my purposes ill just keep it a servercore, large image size doesnt bother me. for your purposes you are correct